wgengine/router/router_linux_test.go

```diff
@@ -606,6 +606,50 @@ func (n *fakeIPTablesRunner) DelSNATRule() error {
 	return nil
 }
 
+func buildMagicsockPortRule(port uint16) string {
+	return fmt.Sprintf("-p udp --dport %v -j ACCEPT", port)
+}
+
+func (n *fakeIPTablesRunner) AddMagicsockPortRule(port uint16, network string) error {
```

Comment:
This is possibly just a me thing - something I like to do for fakes like this is add a doc string explaining what the function actually does in comparison to the real implementation. 

For example, there have been occasions where I've had to implement a fake, but I'm not testing what the fake is doing, but what something else is doing. In those cases, since I'm possibly not implementing the full checks/etc into that functionality, I want to make it clear what the fake is there to do.

In cases where you are using a fake because you are unable to test the real behaviour in CI/on other platforms (e.g. if you are making external API calls to services), it can be good to detail that as well.